fmfexporter/fmf_adapter.py

```diff
@@ -91,7 +91,7 @@ def submit_testcase(self, fmf_testcase: FMFTestCase):
         This method is used to submit a generic FMFTestCase element
         into the external ALM related tool.
         :param fmf_testcase:
-        :return:
+        :return: submitted_testcase
```

Comment:
Revert this as nothing is being returned.

Reply:
removed. (will be in next commit)